pages/report.vue

```diff
@@ -0,0 +1,303 @@
+              us here.
+            </p>
+            <p>
+              This form is intended exclusively for reporting abuse or harmful content to Modrinth
+              staff. For bugs related to specific projects, please use the project's designated
+              Issues link or Discord channel.
+            </p>
+            <p>
+              Your privacy is important to us; rest assured that your identifying information will
+              be kept confidential.
+            </p>
+          </div>
+        </div>
+        <div>
+          <div class="report-info-section">
+            <div class="report-info-item">
+              <label for="report-item">Item Type to Report</label>
+              <DropdownSelect
+                id="report-item"
+                v-model="reportItem"
+                name="report-item"
+                :options="reportItems"
+                :display-name="capitalizeString"
+                :multiple="false"
+                :searchable="false"
+                :show-no-results="false"
+                :show-labels="false"
+                placeholder="Choose report item"
+              />
+            </div>
+            <div class="report-info-item">
+              <label for="report-item-id">Item ID</label>
+              <input
+                id="report-item-id"
+                v-model="reportItemID"
+                type="text"
+                placeholder="ex. project ID"
+                autocomplete="off"
+                :disabled="reportItem === ''"
+              />
+            </div>
+            <div class="report-info-item">
+              <label for="report-type">Reason for Report</label>
+              <DropdownSelect
+                id="report-type"
+                v-model="reportType"
+                name="report-type"
+                :options="reportTypes"
+                :multiple="false"
+                :searchable="false"
+                :show-no-results="false"
+                :show-labels="false"
+                :display-name="capitalizeString"
+                placeholder="Choose report type"
+              />
+            </div>
+          </div>
+        </div>
+        <div class="report-submission-section">
+          <div>
+            <p>
+              Please provide additional context about your report. Include links and images if
+              possible. <strong>Empty reports will be closed.</strong>
+            </p>
+          </div>
+          <MarkdownEditor v-model="reportBody" placeholder="" :on-image-upload="onImageUpload" />
+        </div>
+        <div>
+          <label for="submit-button">
+            <span class="label__title">Submit Report</span>
+          </label>
+          <p>
+            By submitting this report, you agree to our
+            <nuxt-link class="text-link" to="/legal/rules">Rules</nuxt-link> and
+            <nuxt-link class="text-link" to="/legal/terms">Terms of Service</nuxt-link>.
+          </p>
```

Comment:
Unnecessary. People already agree to the ToS by having a Modrinth account. Content rules aren't really applicable to reports since it's not public.
```suggestion
```